apicast/src/configuration.lua

```diff
@@ -435,7 +439,8 @@ function _M.download(endpoint)
   if body and res.status == 200 then
     ngx.log(ngx.DEBUG, 'configuration response received:' .. body)
 
-    local ok, err = _M.decode(body)
+    local ok
```

Comment:
Yeah. I'm not sure how to solve this. If you have several possible error responses in one method, should they have different names? Or reuse them? Guess more reading of official packages needed :)